tests/Api/Admin/CatalogPromotionsTest.php

```diff
@@ -166,9 +174,17 @@ public function it_updates_catalog_promotion(): void
             json_encode([
                 'name' => 'T-Shirts discount',
                 'code' => 'new_code',
+                'actions' => [
+                    [
+                        'type' => CatalogPromotionActionInterface::TYPE_PERCENTAGE_DISCOUNT,
+                        'configuration' => [
+                            'amount' => '40%'
```

Comment:
```suggestion
                            'amount' => 40
```
or
```suggestion
                            'amount' => 0.4
```

I prefer the latter